demo/src/main/java/com/novoda/accessibility/demo/custom_actions/TweetView.java

```diff
@@ -44,23 +43,43 @@ protected void onFinishInflate() {
     }
 
     public void display(String tweet, Listener listener) {
-        Actions actions = createActions(tweet, listener);
-        ActionsAccessibilityDelegate delegate = new ActionsAccessibilityDelegate(getResources(), actions);
-        delegate.setClickLabel(R.string.tweet_actions_usage_hint);
+        MenuItem.OnMenuItemClickListener menuItemClickListener = createMenuItemClickListener(tweet, listener);
+        Menu menu = actionsMenuInflater.inflate(R.menu.tweet_actions, menuItemClickListener);
+        ActionsMenuAccessibilityDelegate delegate = new ActionsMenuAccessibilityDelegate(menu, menuItemClickListener);
+//      TODO: add composable delegate - delegate.setClickLabel(R.string.tweet_actions_usage_hint);
```

Comment:
fully composable delegate not legit - will make one _in the demo_ that wraps this and UsageHints one

Reply:
Would rather just kill this TODO and add an issue.